backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5688150_sys_Fix_HUs_by_setting the correct_locators.sql

```diff
@@ -0,0 +1,22 @@
+select backup_table('m_hu');
+
+with records as (
+    SELECT t.qty,
+           hu.m_hu_id,
+           hu.hustatus,
+           hu.m_locator_id,
+           io.c_doctype_id,
+           t.m_inout_id,
+           iol.m_inoutline_id,
+           iol.m_locator_id AS iol_locator_id,
+           iol.m_product_id
+
+    FROM m_hu hu
+             JOIN m_hu_trace t ON t.m_hu_id = hu.m_hu_id
+             JOIN m_inout io ON io.m_inout_id = t.m_inout_id and io.c_doctype_id=541005 -- repair
+             JOIN m_hu_assignment hua ON hu.m_hu_id = hua.m_hu_id AND hua.ad_table_id = 320
+             JOIN m_inoutline iol ON io.m_inout_id = iol.m_inout_id AND iol.m_inoutline_id = hua.record_id
+   -- WHERE hu.m_locator_id IS NULL
```

Comment:
why this is commented out?
why overriding existing M_HU.M_Locator_ID? ... I would agree to do that but only in case of  customer returns!

Reply:
Yes, because the locator is set wrong for the warranty product as well and other we have a locator set.
And yes, we doing that only for repairs(c_doctype_id=541005 )

Reply:
also, i think this script shall not be here but in customer repo.

Reply:
I will move it